server/dba/seeders/forests.js

```diff
@@ -10,7 +10,9 @@ const forests = [
       org_structure_code: '11-01-04',
       description: 'Idaho Panhandle | Idaho',
       forest_abbr: 'ipnf',
-      wood_cost: 5,
+      wood_cost: 5.00,
+      min_cords: 4,
+      max_cords: 2,
       timezone: 'America/Boise',
```

Comment:
it looks like ipnf's min and max are swapped here?

Reply:
@mtlaney - Another typo here. `max_cords` is supposed to be 12 here